src/main/java/com/patchworkmc/transformer/PatchworkTransformer.java

```diff
@@ -110,9 +122,11 @@ public void accept(String name, byte[] content) {
 		reader.accept(blockSettingsTransformer, ClassReader.EXPAND_FRAMES);
 
 		ClassWriter writer = new ClassWriter(0);
+
 		ModAccessTransformer accessTransformer = new ModAccessTransformer(writer, accessTransformations);
 
-		node.accept(accessTransformer);
+		StringConstantRemapper stringRemapper = new StringConstantRemapper(accessTransformer, new NaiveRemapper(mappings));
```

Comment:
Wouldn't it make more sense to just pass in the NaiveRemapper and potentially reuse it for each remapped mod?